types/apicompat/docs.go

```diff
@@ -0,0 +1,14 @@
+// Package apicompat contains gogo protobuf generated code for protobuf files
+// which are primarily built into the api module with protocgen-go-pulsar.
+// You should generally prefer using the files in the api module and switch
+// over to the new generated code when possible. If you need to reference
+// any of these newer types from proto files which are still generated
+// with gogo proto, you can use a buf.gen.yaml template with buf managed mode
+// to set the correct proto path:
+//  version: v1
+//  managed:
+//   enabled: true
+//   go_package_prefix:
+//     override:
+//       buf.build/cosmos/cosmos-sdk: github.com/cosmos/cosmos-sdk/types/apicompat
+package apicompat
```

Comment:
Not sure if this is the best name. I also thought of calling it `gogocompat` but that isn't great either.

Reply:
this is fine since it won't be here long

Reply:
how about `api_gogo`?